### DISCUSSION TITLE
Neptune 4 Max - Layer shifting issues

### DISCUSSION THREAD

**Discussion by spikeon** [OpenNeptune3D/OpenNept4une#203] [General]
I have done EVERYTHING to try to get this damn printer to stop layer shifting.  I have done the full linear rail conversion, upgraded to OpenNept4une, sacrificed a virgin under the full moon, and even replaced the toolhead with a StealthBurner with an onboard processor.  To be clear, it layer shifted BEFORE any of these changes.  My Neptune 4 Pro runs fine and never layer shifts with similar changes, but my Neptune 4 max will layer shift more often than it succeeds (on large prints).  The shift mostly occurs on the Y axis.  It does it when the belt is tight or when the belt is looser.  I am running the printer with the speed settings from the OpenNept4une Orca Profile. 

It happens so often that I can't print the large items on this printer that I bought the printer for. 

Does anyone have any suggestions on how to diagnose or fix this?  

The only things I can think of that I haven't done are: 
- replace the Y axis motor
- replace the Y axis driver
- replace the mainboard with a better one, like one for a Voron, in case the issue is processor based
- Throw the printer off of a bridge

**Comment by u17194907425** [comment]
It looks like you are asking about a method to find the root cause of layer-shifting. Without knowing that you may find yourself doing unnecessary replacements of the printer parts until it becomes a completely different one.

My 1st reaction to this issue would be to: 
- keep it stock (in your situation — revert it back to stock)
- report the issue to the Elegoo support.

**Comment by linhungsam** [comment]
> I have done EVERYTHING to try to get this damn printer to stop layer shifting. I have done the full linear rail conversion, upgraded to OpenNept4une, sacrificed a virgin under the full moon, and even replaced the toolhead with a StealthBurner with an onboard processor. To be clear, it layer shifted BEFORE any of these changes. My Neptune 4 Pro runs fine and never layer shifts with similar changes, but my Neptune 4 max will layer shift more often than it succeeds (on large prints). The shift mostly occurs on the Y axis. It does it when the belt is tight or when the belt is looser. I am running the printer with the speed settings from the OpenNept4une Orca Profile.
> 
> It happens so often that I can't print the large items on this printer that I bought the printer for.
> 
> Does anyone have any suggestions on how to diagnose or fix this?
> 
> The only things I can think of that I haven't done are:
> 
> * replace the Y axis motor
> * replace the Y axis driver
> * replace the mainboard with a better one, like one for a Voron, in case the issue is processor based
> * Throw the printer off of a bridge

Sounds like you are heavily modified your printer, I will go with option 4, "Throw the printer off of a bridge". Your mods may not work for OpenNept4une as it is designed for Elegoo's original hardware. Your N4 Pro runs fine, it could be that your new components luckily match the N4 Pro OpenNept4une's configuration. Make sure you use the correct configuration for N4 Max or Plus, they have different dimension and motor configuration.

**Comment by spikeon** [comment]
So, I have it printing reliably now; it was either a faulty or overwhelmed Y axis motor.  I replaced it with a larger NEMA 17 and now my prints are great.Sent from my iPhoneOn Sep 5, 2024, at 12:44 PM, Slamurai ***@***.***> wrote:﻿

I have done EVERYTHING to try to get this damn printer to stop layer shifting. I have done the full linear rail conversion, upgraded to OpenNept4une, sacrificed a virgin under the full moon, and even replaced the toolhead with a StealthBurner with an onboard processor. To be clear, it layer shifted BEFORE any of these changes. My Neptune 4 Pro runs fine and never layer shifts with similar changes, but my Neptune 4 max will layer shift more often than it succeeds (on large prints). The shift mostly occurs on the Y axis. It does it when the belt is tight or when the belt is looser. I am running the printer with the speed settings from the OpenNept4une Orca Profile.
It happens so often that I can't print the large items on this printer that I bought the printer for.
Does anyone have any suggestions on how to diagnose or fix this?
The only things I can think of that I haven't done are:

replace the Y axis motor
replace the Y axis driver
replace the mainboard with a better one, like one for a Voron, in case the issue is processor based
Throw the printer off of a bridge


Sounds like you are heavily modified your printer, I will go with option 4, "Throw the printer off of a bridge". Your mods may not work for OpenNept4une as it is designed for Elegoo's original hardware. Your N4 Pro runs fine, it could be that your new components luckily match the N4 Pro OpenNept4une's configuration. Make sure you use the correct configuration for N4 Max or Plus, they have different dimension and motor configuration.

**Reply by autocannibal** [reply]
could you drop a link to that motor?

**Reply by spikeon** [reply]
https://west3d.com/products/ldo-motors-42sth60-2004ac-nema-17-stepper-motor-supermac?variant=44217343475924

Note:  i had to add some foot extensions to raise the printer

**Reply by autocannibal** [reply]
Thank you, Identical specs to the stock motor in printer.cfg?

**Reply by spikeon** [reply]
I think so, it's been a minute and that printer has since died in a very unrelated way.

**Reply by autocannibal** [reply]
thats ominous... you didnt throw it off a bridge did you?

**Reply by spikeon** [reply]
No, i wasn't careful when putting the faceplate of my Stealthburner back on and the pins went in the wrong holes.  Burnt out the entire toolhead board, fan, led's, U2C, and it must've partially damaged the usb port on the mainboard because while the printer turns on it can no-longer connect to anything via usb.  So if I want to keep using it i need to replace the mainboard.  At that point I might as well just put in a much better one that already has canbus.  So it's in the basement in pieces for now.

**Comment by gillytak** [comment]
I have same issueds spent way too much time trying to fix it so my Neptune 4MAX is about to go to a paupers grave i bought a bamboo H2D S and Dammm it's awesome speed is great everything is awesome about it best 2.4k i ever spent.

**Comment by JesseJamesMcGuire** [comment]
My fix for layer shifts were to tighten the belts and slow the print speed way down.    Just because it can print fast doesnt mean it should..
